packages/dds/sequence/api-report/sequence.api.md

```diff
@@ -159,8 +159,6 @@ export interface IIntervalCollection<TInterval extends ISerializableInterval> ex
     // (undocumented)
     CreateForwardIteratorWithStartPosition(startPosition: number): Iterator<TInterval>;
     detachIndex(index: IntervalIndex<TInterval>): boolean;
-    // (undocumented)
-    findOverlappingIntervals(startPosition: number, endPosition: number): TInterval[];
```

Comment:
public apis cannot be removed in main. they must first be deprecated in main, and then removed in next